cluster-install/install.sh

```diff
@@ -2532,11 +2542,34 @@ if trident_will_be_installed_or_modified; then
         if components_include_trident; then
             # Trident upgrade (includes operator upgrade if needed)
             if trident_image_needs_upgraded; then
-                if config_trident_image_is_custom || prompt_user_yes_no "Would you like to upgrade Trident?"; then
-                    step_generate_torc_patch "$_EXISTING_TORC_NAME" "$(get_config_trident_image)" "" "" "$(get_config_trident_autosupport_image)"
-                    trident_operator_image_needs_upgraded && step_generate_trident_operator_patch
-                else
-                    loginfo "Trident will not be upgraded."
+                # if trident version < 23.10
+                if version_higher_or_equal "23.07" "$_EXISTING_TRIDENT_VERSION"; then
+                    logwarn "Your Trident installation is at version 23.07, while the lowest required version to enable ACP is 23.10."
```

Comment:
Could the trident version here be lower than 23.07? Might need to put in the actual version in the log:
```suggestion
                    logwarn "Your Trident installation is at version $_EXISTING_TRIDENT_VERSION, while the lowest required version to enable ACP is 23.10."
```